convert2rhel/main.py

```diff
@@ -167,12 +167,11 @@ def main_locked():
             subscription.update_rhsm_custom_facts()
 
         rollback_changes()
+        print_info_after_rollback(pre_conversion_results, True)
+
+        if backup.backup_control.rollback_failed:
+            return 1
 
-        report.summary(
-            pre_conversion_results,
-            include_all_reports=True,
-            disable_colors=logger_module.should_disable_color_output(),
-        )
         return 0
```

Comment:
We could simply `return backup.backup_control.rollback_failed`

Reply:
I asked @hosekadam to turn return explicitly here to make it easier to look at the return value and see what code we expect. 

But, regardless, with https://issues.redhat.com/browse/RHELC-1276, I'm planning to introduce an enum to match the return codes we (will) have to figure out what does 0/1/2/... means when returning.